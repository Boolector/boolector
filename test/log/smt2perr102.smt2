(declare-fun a () (Array (_ BitVec 16) (_ BitVec 1)))
(assert (store (_ bv0 32) a (_ bv0 32)))
