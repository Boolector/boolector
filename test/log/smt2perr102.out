log/smt2perr102.smt2:2:16: first argument of 'store' is not an array
